simulation-runner/config/scenarios/README.md

```diff
@@ -1 +1,48 @@
-We are now automatically generating the scenario config files at runtime.  This file is just here as a placeholder to ensure that this directory exists at runtime, so that the config files can be written here.
+## Scenario configuration files
+
+This directory contains "scenario" configuration files.  These files are used to
+describe an actual gatling run; they reference one or more
+["node" config files](../nodes) to determine which agent recordings to play
+back, and they also set up the parameters for each type of agent/node (e.g. how
+many instances of that node to simulate simultaneously, etc.).
+
+### File format
+
+Here is an example "scenario" configuration file:
+
+```json
+{
+    "run_description": "'medium' role from perf control repo",
+    "nodes": [
+        {
+            "node_config": "PECouchPerfMedium.json",
+            "num_instances": 1250,
+            "ramp_up_duration_seconds": 1800,
+            "num_repetitions": 4,
+            "sleep_duration_seconds": 1800
+        }
+    ]
+}
+```
+
+Here is a breakdown of the fields:
+
+* `run_description`: a string with a basic description of the scenario.  Will show up in a few places, e.g. the
+  Gatling report HTML, but can be anything you like.
+* `nodes`: an array of one or more types of nodes/puppet agents to simulate.  Each node/agent entry is a map with the
+  following keys:
+** `node_config`: this is the name of the node config json file in [the `../nodes` directory](../nodes).  This ends up
```

Comment:
The ** sections aren't rendering as indented sections for me when I view in a browser.  You might take a look at that - https://github.com/cprice404/gatling-puppet-load-test/blob/maint/master/SERVER-1474-update-docs/simulation-runner/config/scenarios/README.md.